examples/fiber/main.go

```diff
@@ -0,0 +1,125 @@
+package main
+
+import (
+	"encoding/json"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/airbrake/gobrake/v5"
+	fiberbrake "github.com/airbrake/gobrake/v5/fiber"
+	"github.com/gofiber/fiber/v2"
+)
+
+var ProjectId int64 = 999999                               // Insert your Project Id here
+var ProjectKey string = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx" // Insert your Project Key here
+
+var notifier = gobrake.NewNotifierWithOptions(&gobrake.NotifierOptions{
+	ProjectId:   ProjectId,
+	ProjectKey:  ProjectKey,
+	Environment: "production",
+})
+
+var availableLocations = []string{
+	"austin",
+	"pune",
+	"santabarbara",
+}
+
+// Define models
+
+type Date struct {
+	Date int64 `json:"date,omitempty"`
+}
+
+type WeatherInfo struct {
+	Lat            float32       `json:"lat,omitempty"`
+	Lon            float32       `json:"lon,omitempty"`
+	Timezone       string        `json:"timezone,omitempty"`
+	TimezoneOffset float32       `json:"timezone_offset,omitempty"`
+	Current        interface{}   `json:"current,omitempty"`
+	Minutely       []interface{} `json:"minutely,omitempty"`
+	Hourly         []interface{} `json:"hourly,omitempty"`
+	Daily          []interface{} `json:"daily,omitempty"`
+}
+
+type Error struct {
+	Error string `json:"error,omitempty"`
+}
+
+func main() {
```

Comment:
I see one example of where `notifier` is used in the example app for error handling (in `getWeather`)? Should there be a `defer notifier.NotifyOnPanic()` in `main`? Should there be some error monitoring in `TokenAuthMiddleware`?

The example should also have some documentation that explains how to trigger an error so that the user can check for that error in Airbrake. For example, what if `availableLocations` could include another location name, e.g. `newyork` but there is no JSON file for that location. That would trigger an error from `os.Open(weatherInfoFilePath)` and Airbrake would be notified.

Reply:
Should there be a `defer notifier.NotifyOnPanic()` in main?